### PR TITLE
Update/230904

### PR DESCRIPTION
**Pull request by maxb-hystax**

### CODE REVIEW

File: docker_images/grafana_nginx/htpasswd [hunk 1]
```diff
@@ -0,0 +1,2 @@
+grafana:$apr1$u8e6RJMa$ThJKHrclQ2EZCz3gmUcKT/
```
Comment (tguisep) [inline docker_images/grafana_nginx/htpasswd:1]: Hashed committed pwd

Comment (maxb-hystax) [reply docker_images/grafana_nginx/htpasswd:1]: let's leave it as is for now to finish with fully buildable code.
We have plan to move htpasswd file generation to the runtime (from overlay file) once we update deployment charts

File: jira_ui/server/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const app = express();
+
+app.use(
+  "/jira_ui",
+  express.static(path.join(process.env.UI_BUILD_PATH, "build"))
+);
+
+app.get("/*", (req, res) => {
+  res.sendFile(path.join(process.env.UI_BUILD_PATH, "build", "index.html"));
+});
```
[the remark this answers is not in the document]
Comment (maxb-hystax) [reply jira_ui/server/index.js:17]: Filed separate ticket to close it.

**Review by nexusriot — APPROVED** [review]

File: docker_images/grafana_nginx/htpasswd (hunk 1 above)
Comment (maxb-hystax) [reply docker_images/grafana_nginx/htpasswd:1]: Filed https://github.com/hystax/optscale/issues/132

**Pull request merged**